addons/io_scene_gltf2/__init__.py

```diff
@@ -336,6 +334,16 @@ def invoke(self, context, event):
                 self.report({"ERROR"}, "Loading export settings failed. Removed corrupted settings")
                 del context.scene[self.scene_key]
 
+        self.user_extensions = {}
+        for addon_name in bpy.context.preferences.addons.keys():
+            try:
+                # Every user-extension would have a top-level class called Gltf2UserExt
+                extension_ctor = getattr(sys.modules[addon_name], "glTF2UserExtension")
```

Comment:
You don't need the `getattr` stuff. The except block will catch any exception if it isn't there.